validator/sawtooth_validator/journal/block_store.py

```diff
@@ -130,10 +130,9 @@ def get_predecessor_iter(self, starting_block=None):
         Returns:
             An iterator
         """
-        if not starting_block:
-            return _BlockPredecessorIterator(self, start_block=self.chain_head)
-        else:
-            return _BlockPredecessorIterator(self, start_block=starting_block)
+        start_block = starting_block if starting_block else self.chain_head
```

Comment:
IMO, introducing another variable like this adds an unnecessary level of indirection. I prefer something like the following:
```
if not starting_block:
    return _BlockPredecessorIterator(self, start_block=self.chain_head)

return _BlockPredecessorIterator(self, start_block=starting_block)
```